presto-hive/src/main/java/com/facebook/presto/hive/s3/PrestoS3SelectClient.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.s3;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.SelectObjectContentEventVisitor;
+import com.amazonaws.services.s3.model.SelectObjectContentRequest;
+import com.amazonaws.services.s3.model.SelectObjectContentResult;
+import com.facebook.presto.hive.HiveClientConfig;
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static com.amazonaws.services.s3.model.SelectObjectContentEvent.EndEvent;
+import static java.util.Objects.requireNonNull;
+
+public class PrestoS3SelectClient
+        implements Closeable
+{
+    private final AmazonS3 s3Client;
+    private boolean isRequestComplete;
```

Comment:
The `is` prefix is normally only used for getters. The field itself should be named without it, so just `requestComplete`

Reply:
makes sense, refactored